src/sentry/tasks/integrations/migrate_issues.py

```diff
@@ -0,0 +1,53 @@
+from django.db import IntegrityError, transaction
+from rest_framework.response import Response
+
+from sentry.models import ExternalIssue, Group, GroupLink, GroupMeta, Integration
+from sentry.tasks.base import instrumented_task, retry
+from sentry.tasks.integrations import logger
+
+
+@instrumented_task(
+    name="sentry.tasks.integrations.migrate_issues",
+    queue="integrations",
+    default_retry_delay=60 * 5,
+    max_retries=5,
+)
+@retry(exclude=(Integration.DoesNotExist))
+def migrate_issues(
+    integration_id: int, organization_id: int, project_id: int, plugin_slug: str
+) -> None:
```

Comment:
I think it might be better to have this task just use the organization, integration and plugin, instead of scoping it to the project. It'd mean we'd only fire one task per organization instead of one for every project an organization has.

You would have to move it outside the for loops of `integration.migrate_issues` for both Jira and Jira Server

Reply:
You could also instead have this task accept a list of projects so you can call
```python
Group.objects.filter(
  project__in=projects,
  project__organization_id=organization_id
)
```